blurring_as_a_service/metadata_pipeline/utils/coco_to_yolo_converter.py

```diff
@@ -1,12 +1,14 @@
 import json
 
 
-def _update_category(annotation):
-    annotation["category_id"] = annotation["category_id"] - 1
-    return annotation
-
-
 class CocoToYoloConverter:
+    """
+    Converts a COCO annotation dataset to a YOLOv5 format.
+    - The bbox will remain the same format;
```

Comment:
The bbox changes from `x_min, y_min, width, height` to `x_center, y_center, width, height`.